### PR TITLE
[Feature-New-Edged]rename fake kubeclient with kubeclient bridge

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
Signed-off-by: Shelley-BaoYue <baoyue2@huawei.com>

**What type of PR is this?**

/kind feature

#4157 

**What this PR does / why we need it**:
Rename `fake` directory in edged with `kubeclientbridge`, its more suitable and easy to understand.

### CODE REVIEW

File: edge/pkg/edged/edged.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ import (
 	edgecadvisor "github.com/kubeedge/kubeedge/edge/pkg/edged/cadvisor"
 	"github.com/kubeedge/kubeedge/edge/pkg/edged/clcm"
 	edgedconfig "github.com/kubeedge/kubeedge/edge/pkg/edged/config"
-	fakekube "github.com/kubeedge/kubeedge/edge/pkg/edged/fake"
+	fakekube "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge"
```
Comment (wackxu) [inline edge/pkg/edged/edged.go:110]: suggest	
`clientbridge "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge"`

File: edge/pkg/edged/kubeclientbridge/typed/core/v1/core_client_bridge.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package v1
```
Comment (wackxu) [inline edge/pkg/edged/kubeclientbridge/typed/core/v1/core_client_bridge.go:1]: add copyright for the file

File: edge/pkg/edged/edged.go (hunk 1 above)
Comment (Shelley-BaoYue) [reply edge/pkg/edged/edged.go]: > suggest `clientbridge "github.com/kubeedge/kubeedge/edge/pkg/edged/kubeclientbridge"`

Good suggestion！I've already modify it.

File: edge/pkg/edged/kubeclientbridge/typed/core/v1/core_client_bridge.go (hunk 2 above)
Comment (Shelley-BaoYue) [reply edge/pkg/edged/kubeclientbridge/typed/core/v1/core_client_bridge.go:24]: > add copyright for the file

Already done, and alse with other files.

File: edge/pkg/edged/kubeclientbridge/clientset_bridge.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```
[the remark this answers is not in the document]
Comment (Shelley-BaoYue) [reply edge/pkg/edged/kubeclientbridge/clientset_bridge.go:2]: just rename the file, the content has no changes. 😄

File: edge/pkg/edged/kubeclientbridge/typed/core/v1/configmap_bridge.go [hunk 4]
```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```
[the remark this answers is not in the document]
Comment (Shelley-BaoYue) [reply edge/pkg/edged/kubeclientbridge/typed/core/v1/configmap_bridge.go:2]: ditto

**Review by vincentgoat — COMMENTED** [review]
lgtm

**Review by fisherxu — COMMENTED** [review]
Since we already have the `bridge` in `kubeclientbridge`, so the file name in sub-dir can remove the `bridge`?

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**